src/main/java/com/arpnetworking/metrics/mad/parsers/CollectdJsonToRecordParser.java

```diff
@@ -87,7 +87,8 @@
                 final DefaultRecord.Builder builder = new DefaultRecord.Builder()
                         .setId(UUID.randomUUID().toString())
                         .setTime(record.getTime())
-                        .setAnnotations(ImmutableMap.copyOf(metricTags));
+                        .setAnnotations(ImmutableMap.copyOf(metricTags))
```

Comment:
Is there a reason to keep these as annotations, I don't believe there were any tags in CD that were not annotations.